api/src/api/notebooks.ts

```diff
@@ -244,14 +245,42 @@ api.get(
     if (!req.user || !userHasPermission(req.user, req.params.id, 'read')) {
       throw new Exceptions.UnauthorizedException();
     }
-    const records = await getNotebookRecords(req.params.id, {
+    const token = {
       roles: req.user.roles,
       server: slugify(CONDUCTOR_INSTANCE_NAME),
       username: req.user.user_id,
       // Five minutes from now
       exp: Date.now() + 1000 * 60 * 5,
-    });
+    };
+    const records = await getRecordsWithRegex(token, req.params.id, '.*', true);
```

Comment:
get records with regex with .* is probably the same as get all records? Isn't there already that function?

Reply:
Not one that returns the kind of record that we want...